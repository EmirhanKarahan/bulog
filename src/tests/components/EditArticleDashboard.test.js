import React from "react";
import { shallow } from "enzyme";
import EditArticleDashboardPage from "../../components/EditArticleDashboardPage"

test("should render EditArticleDashboardPage correctly", ()=>{
    const wrapper = shallow(<EditArticleDashboardPage />)
    expect(wrapper).toMatchSnapshot();
})