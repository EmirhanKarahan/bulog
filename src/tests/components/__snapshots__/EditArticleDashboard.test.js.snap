// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render EditArticleDashboardPage correctly 1`] = `
<Connect(ArticleList)
  editable={true}
/>
`;
